16x16.js/sketch.js

```diff
@@ -33,22 +34,22 @@ function setup() {
   //useMode("Random Mode")
 }
 
-const updateGridUnit = () => (u = Math.min(windowWidth, windowHeight) / 25)
-function draw() {
-  updateGridUnit()
+// returns value alternating based on time
+const blinking = (on, off) => (millis() % 1000) > 500 ? on : off
 
-  // update unit of measurement
-  // shortest width divided by 25 (leaving a border of 2 on each side around grid)
+// shortest width divided by 25 (leaving a border of 2 on each side around grid)
+const unitOfOne = () => Math.min(windowWidth, windowHeight) / 25
+const unitOf = scale => unitOfOne() * scale
 
 
+function draw() {
   background(0)
-  renderGrid(u * 0.5, u * 0.5, u * 16, u * 16)
+  renderGrid(windowWidth / 2 - unitOf(8), windowHeight / 2 - unitOf(8), unitOf(16), unitOf(16))
```

Comment:
I have moved the coordinates and dimensions back here to keep ther render grid function a little cleaner.